src/js/vendor/highlight.bundle.js

```diff
@@ -1,8 +1,8 @@
 ;(function () {
   'use strict'
 
-  var hljs = require('highlight.js/lib/core')
-  var hljsDefineCairo = require('highlightjs-cairo')  // Require the highlightjs-cairo module
+  const hljs = require('highlight.js');
```

Comment:
Since highlighting isn't working for any of the languages, I'm guessing there could be an issue with how this line is defined.  Can you try the following from [this example](https://github.com/highlightjs/highlight.js/tree/9-18-stable?tab=readme-ov-file#nodejs)?
```suggestion
  const hljs = require('highlight.js/lib/highlight.js');
```